server/tests/advertiser.js

```diff
@@ -443,4 +444,184 @@ describe('Advertiser', () => {
         done();
       });
   });
+  it('Should return status code 200', (done) => {
+    chai.request(app)
+      .get('/api/v1/advertiser/announcements')
+      .set('Authorization', `Bearer ${token}`)
+      .end((err, res) => {
+        expect(res.body).to.have.property('status');
+        expect(res.status).to.equal(200);
+        expect(res.body).to.have.property('data');
+        expect(res.body.data[0]).to.have.property('id');
+        expect(res.body.data[0].id).to.be.a('number');
+        expect(res.body.data[0]).to.have.property('title');
+        expect(res.body.data[0]).to.have.property('description');
+        expect(res.body.data[0]).to.have.property('startdate');
+        expect(res.body.data[0]).to.have.property('enddate');
+        expect(res.body.data[0]).to.have.property('status');
+        expect(res.body.data[0]).to.have.property('owner');
+        expect(res.body.data[0].owner).to.be.a('number');
+        expect(res.body.data[0]).to.have.property('createdon');
```

Comment:
since you are required to use es6 it would be good if you use destructuring then expect from all the created variables

Reply:
Noted.